src/browser/components/ManualLink.jsx

```diff
@@ -20,13 +20,25 @@
 
 import React from 'react'
 import { connect } from 'react-redux'
+import semver from 'semver'
 
 import { getVersion } from 'shared/modules/dbMeta/dbMetaDuck'
 import { formatDocVersion } from 'browser/modules/Sidebar/Documents'
 
-export function ManualLink({ chapter, page, children, neo4jVersion }) {
+export function ManualLink({
```

Comment:
this is more of a general question as I need to understand too - do you know why some files use arrow funcs and others function declaration syntax? I realise there are differences when it comes to this (like `this`, `arguments` and usage as a constructor) but beyond that Im not sure why we mix the two or is it not deemed to be an issue? @oskarhane maybe you can shed more light on this?

Reply:
Good observation @jk05. This is just due to personal preference and that we don't see it as an issue.
The main goal is for the code to be readable and easy to understand. I think in many cases there might just be code that's a bit older.
Feel free to modernize code you come across with your changes.
I see it as continuous maintenance (like moving away from deprecated lifecycle methods, porting classes to hooks etc) as I com across it.